### DISCUSSION TITLE
Problem when open the maps in IOS

### DISCUSSION THREAD

**Discussion by brunonunestx** [developmentseed/lonboard#709] [General]
I create the maps and generate links to access it, but when I open it in IOS it don't work, the maps crashes. I don't know how to fix it. I test in Safari and Google Chrome, and I get the same problem. Somebody can help me?

I opened the Google Chrome Dev Tools and the console is the following.

**Comment by kylebarron** [comment]
I haven't tried to use lonboard-based maps on iOS. On iOS, all browsers are provided by the default webkit-based renderer. iOS WebKit is usually behind the features available in recent desktop Chrome or Safari versions (especially if you have an old iOS version).

Your screenshot also doesn't show any errors; those are just logs and warning messages.

You also probably shouldn't share your AWS keys in public.

**Reply by brunonunestx** [reply]
Thank you!
I forgot that the AWS keys was visible, can you delete this discussion, please? I edit but it is visible in the edited histories and I try delete this but I didn't get it.

**Reply by kylebarron** [reply]
This has been online for 17 hours already, and you should consider your keys exposed. Instead of deleting this discussion you should go into the AWS console and regenerate those keys.
